system-test/spanner.ts

```diff
@@ -3560,24 +3525,13 @@ describe('Spanner', () => {
           backup => backup.formattedName_ === googleSqlBackup1.formattedName_,
         ),
       );
-      assert.ok(
-        page3.find(
-          backup => backup.formattedName_ === googleSqlBackup2.formattedName_,
-        ),
-      );
       if (!IS_EMULATOR_ENABLED && !SKIP_POSTGRESQL_BACKUP_TESTS) {
```

Comment:
can we remove this condition (!IS_EMULATOR_ENABLED) from here? we anyway have below check in before block
```
if (IS_EMULATOR_ENABLED) {
  this.skip();
}
```
hence, if this test is running that means emulator is not enabled?